src/api/app/views/webui2/shared/_flag_popover.html.haml

```diff
@@ -0,0 +1,28 @@
+- if flag.new_record?
```

Comment:
Just to understand it: How can it be a new record?

Reply:
[nitpick] The only difference between the if / else are the URLs, right? Can we not just compute the URL in the if/else or somewhere else and then the partial just has the presentation with whatever link we computed?

Reply:
> Just to understand it: How can it be a new record?

Not all the flags are in the DB, the majority is calculated base on the default value for the flag type or is based on the project one..., so for showing it we always create a new record flag if it is not in the DB.

Reply:
> [nitpick] The only difference between the if / else are the URLs, right? Can we not just compute the URL in the if/else or somewhere else and then the partial just has the presentation with whatever link we computed?

Not only the url really, also the icon and the text are different

Reply:
Smart :+1:

Reply:
> Not only the url really, also the http method, the icon and the text are different
 
Now I see it, but still very similar. But fine for me leaving it like it is...